src/pykeen/triples/triples_factory.py

```diff
@@ -361,6 +393,11 @@ def __init__(
         self.metadata = metadata
         self.relation_inverter = relation_inverter_resolver.make(query=None)
 
+    @classmethod
+    def new(cls, **kwargs: Any) -> Self:
```

Comment:
is there a place where new() does something other than calling cls()?

Reply:
answer in https://github.com/pykeen/pykeen/pull/1416#discussion_r1947965982